### PR TITLE
Fix help message

### PR DESCRIPTION
**Pull request by vipkr**
This pull request contains following changes:

- Add missing -f/-F option in README.
- Start first letter of help message with a capital letter and end with a period punctuation to make things consistent with other option messages.

/cc @vathpela

### CODE REVIEW

File: src/efibootmgr.8.in [hunk 1]
```diff
@@ -26,26 +26,26 @@ non-volatile variables through
 The following is a list of options accepted by efibootmgr:
 .TP
 \fB-a | --active\fR
-Sets bootnum active
+Sets bootnum active.
```
Comment (paulmenzel) [inline src/efibootmgr.8.in:29]: Below also the imperative mood is used, maybe this could be fixed as well?

**Review by paulmenzel — APPROVED** [review]
Thank you for sending patches. I am not sure about adding the dot/period at the end of the “imperative mood statements”. I am not sure about the English grammar, but probably you are right, as when the statement gets longer, the period/dot seems to be the right way to do it.

File: src/efibootmgr.8.in (hunk 1 above)
Comment (finn0) [reply src/efibootmgr.8.in:29]: Didn't get it? Should I remove period punctuation from imperative mood sentence? Or I've miss some imperative mood sentence which doesn't end with period.

File: src/efibootmgr.c [hunk 2]
```diff
@@ -1350,22 +1350,22 @@ usage()
...
+	printf("\t-e | --edd [1|3]      Force boot entries to be created using EDD 1.0 or 3.0 info.\n");
+	printf("\t-E | --device num     EDD 1.0 device number (defaults to 0x80).\n");
+	printf("\t     --full-dev-path  Use a full device path.\n");
+	printf("\t     --file-dev-path  Use an abbreviated File() device path.\n");
+	printf("\t-f | --reconnect      Re-connect devices after driver is loaded.\n");
+	printf("\t-F | --no-reconnect   Do not re-connect devices after driver is loaded.\n");
+	printf("\t-g | --gpt            Force disk with invalid PMBR to be treated as GPT.\n");
+	printf("\t-i | --iface name     Create a netboot entry for the named interface.\n");
 #if 0
 	printf("\t     --ip-addr <local>,<remote>	set local and remote IP addresses\n");
```
Comment (finn0) [inline src/efibootmgr.c:1370]: @paulmenzel As far as I can see, I've missed these sentences, if you want to I can make changes here, also.

File: src/efibootmgr.8.in (hunk 1 above)
Comment (paulmenzel) [reply src/efibootmgr.8.in:29]: Sorry for the confusion. *Sets* is not imperative mood – *Set* would be.

File: src/efibootmgr.c (hunk 2 above)
Comment (paulmenzel) [reply src/efibootmgr.c:1370]: Yes, that’d be great for consistency.

PS: Please note, I am just a drive-by contributor, and I have not the permission to accept merge/pull requests.

Comment (finn0) [reply src/efibootmgr.c:1370]: Okay, I'll make all suggested changes.

**Comment by martinezjavier** [comment]
Most options help messages are not capitalized and don't have a period at the end of the sentence.

I would do the opposite and make all of them use small letters without a period at the end.

**Comment by frozencemetery** [comment]
Closing because stale.  If still needed, please open a new PR, rebased, with changes requested from review.

**Comment by vipkr** [comment]
Sorry, I keep forgetting to update MR with requested reviews.

> Closing because stale.

Github doesn't notify when a merge request marked as a draft. :(

> If still needed, please open a new PR, rebased, with changes requested from review.

Robert has already created a new MR for this, thanks for that. :)

**Pull request closed without merging**